### PR TITLE
Added placeholder for March's event

### PR DESCRIPTION
**Pull request by omgmog**
Filler until more info is available! It'll be interesting to see if the missing data breaks things too 🤷

This should probably not be merged until it's got the necessary info filled in.

@benfoxall or anybody want to add the missing details?

### CODE REVIEW

File: src/events/2021-03.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+date: 2021-03-10T19:00:00Z
+description: 
+speakers:
+ - who: Max
+   twitter: omgmog
+   what: The Pico-8 virtual console
+ - who: Mike
+   twitter: 
+   what:
```
Comment (benfoxbotica) [inline src/events/2021-03.md]: Tell us your name, @mikerhyssmith!

Comment (mikerhyssmith) [reply src/events/2021-03.md]: Hi @benfoxall ! You got my name spot on 😂 

```suggestion
   twitter: mikerhyssmith
   what:  Backstage and onboarding new developers
```

Comment (benfoxall) [reply src/events/2021-03.md]: Oh man 🤦  I really need some plugin that tells me "You're @benfoxbotica"

**Review by benfoxall — APPROVED** [review]
@omgmog, that's Mike's deets and a description in 👍 

Would this be robust to put in future events?!

File: src/events/2021-03.md (hunk 1 above)
Comment (omgmog) [reply src/events/2021-03.md]: @benfoxall You're @benfoxbotica

**Comment by omgmog** [comment]
Looks good to me, and ready to merge (I'm a lowly mortal and I'm unable to merge)

> @omgmog, that's Mike's deets and a description in 👍
> 
> Would this be robust to put in future events?!

Robust enough that we can put the next event in, so the home will show the upcoming event + previous events.

I wouldn't recommend putting further ahead than that as it just takes the _most futurist event_ and shows it on home :shrug:

**Comment by benfoxall** [comment]
Ah, sorry @omgmog.

I've rectified things! You should be able to merge people's PRs, or push straight to `main` if you want (feel free to do either!)

image

**Pull request merged**